addons/test_mail/__manifest__.py

```diff
@@ -11,6 +11,7 @@
 tests independently to functional aspects of other models. """,
     'depends': [
         'mail',
+        'calendar',
```

Comment:
Why is this needed actually?

Reply:
forgot to remove it.
When `calendar` is installed the test_mail tag `:TestServerActionsEmail` needs it.